### PR TITLE
Update the Code of Conduct

### PR DESCRIPTION
**Pull request by alexcoman**

### CODE REVIEW

File: cod.md [hunk 1]
```diff
@@ -25,23 +26,109 @@ No subtle -isms. Our last social rule bans subtle racism, sexism, homophobia, tr
...
-####Consequences
+##### Consequences
 
 Participants asked to stop any harassing behavior are expected to comply immediately.
 
 If a participant engages in harassing behavior, the organizers may take any action they deem appropriate, up to and including expulsion from all OSCamp spaces and events and identification of the participant as a harasser to other OSCamp members or the general public.
+<br>
+
+
+##### Hacker Manifesto
```
Comment (citeaalexandru) [inline cod.md:57]: As adauga ceva introductiv pentru manifest: "We encourage goodwill and a prosperous atitude towards learning. We encourage you to be different. Or be like everybody else." (ultimele doua propozitii in spiritul manifestului)

**Comment by citeaalexandru** [comment]
LGTM

File: cod.md (hunk 1 above)
Comment (alexcoman) [reply cod.md:57]: Super :) Adaug imediat.

File: cod.md [hunk 2]
```diff
@@ -25,23 +26,115 @@ No subtle -isms. Our last social rule bans subtle racism, sexism, homophobia, tr
...
+
+We encourage goodwill and a prosperous atitude towards learning.
```
Comment (TeonaRusu) [inline cod.md:59]: Change atitude to attitude

**Pull request merged**